### ISSUE TITLE
A minior issue about the results on Leaderboard

### ISSUE THREAD

**Issue by boyugou** [google-research/android_world#193]
Hi, I'm the author of UGround ([Navigating the Digital World as Humans Do: Universal Visual Grounding for GUI Agents](https://osu-nlp-group.github.io/UGround/)). It has been a long time after the nice discussions with you guys when I was evaluating on AndroidWorld for UGround.

<img width="1221" alt="image" src="https://github.com/user-attachments/assets/fd601288-568e-4d95-b1ab-92ed4ea2798d">

I'm just wondering if it possible to change the release data of UGround's results to [2024/08/06](https://osu-nlp-group.github.io/UGround/#:~:text=Preprint%20is%20arXived.-,2024/08/06,-%3A%20Website%20is%20live) instead o fOct, since we did release all the results and the first manuscript at that time. And We haven't change the experiments and results after that. Additionally, many works after UGround appeared to notice UGround and had some nice further work.

And btw, it might make more sense to make the columns more granular (given that there are some discussions around modular designs like [SeeAct-V](https://osu-nlp-group.github.io/UGround/#:~:text=SeeAct%2DV%20and%20UGround)), or change UGround's model to GPT-4o+UGround. Otherwise, I feel it a bit unfair to your results (e2e GPT).

**Comment by crawles** [comment]
Yes will be happy to change the date! I think I just quickly looked at the arXiv PDF marker :)

I am not sure I fully understand what specific suggestions re: making it more granular. Definitely happy to update the model name as you suggest

will make these changes later today

**Comment by crawles** [comment]
I have updated the sheet please reopen if you have other comments
